interop/alts/server/server.go

```diff
@@ -30,6 +31,10 @@ import (
 	testpb "google.golang.org/grpc/interop/grpc_testing"
 )
 
+const (
+	udsAddrPrefix = "unix:"
+)
+
 var (
 	hsAddr     = flag.String("alts_handshaker_service_address", "", "ALTS handshaker gRPC service address")
 	serverAddr = flag.String("server_address", ":8080", "The port on which the server is listening")
```

Comment:
The usage string is not accurate. It's not a port.

Reply:
Fixed the description.